lib/pyspark_startup.py

```diff
@@ -0,0 +1,46 @@
+import findspark
+findspark.init()
+
+from pyspark.sql import SparkSession, DataFrame
```

Comment:
Może warto dać jakiś alias np. spDataFrame, pandasowy DataFrame nazywa się tak samo, może to robić problemy w przyszłości

Reply:
Jeżeli nie mam w tym nic poprawiać to proszę zamknąć konwersację.